pkg/kube/pod.go

```diff
@@ -36,35 +38,44 @@ type PodOptions struct {
 	Command            []string
 	Volumes            map[string]string
 	ServiceAccountName string
+	PodOverride        sp.JSONMap
 }
 
 // CreatePod creates a pod with a single container based on the specified image
 func CreatePod(ctx context.Context, cli kubernetes.Interface, opts *PodOptions) (*v1.Pod, error) {
 	volumeMounts, podVolumes := createVolumeSpecs(opts.Volumes)
+	defaultSpecs := v1.PodSpec{
```

Comment:
Minor - just call this `podSpecs` given that it is no longer "default" after line 66